src/components/Nav.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+// import { Link } from 'react-router-dom';
+import './Nav.css';
+
+export default function Nav() {
+  return (
+    <div className="header">
+      <h1 className="title">Math Magicians</h1>
+      <nav>
+        <ul>
+          <li>
+            <a href="/" className="nav-link">Home</a>
+          </li>
+          <li>
+            <a href="/calculator" className="nav-link">Calculator</a>
+          </li>
+          <li>
+            <a href="/quote" className="nav-link">Quote</a>
+          </li>
+        </ul>
+        {/* <Link to="/" className="nav-link">
+      Home |
+    </Link>
+    <Link to="/calculator" className="nav-link">
+      Calculator |
+    </Link>
+    <Link to="/quote" className="nav-link">
+      Quote
+    </Link> */}
```

Comment:
- [ ] It'll be good not to have commented-on code in your codebase as it makes your code unprofessional. It's a good convention to have our codebase clean and professional. Kindly remove all commented unused codes 👍🏽